src/main/java/com/microsoft/sqlserver/jdbc/ISQLServerConnection.java

```diff
@@ -343,4 +343,19 @@ public CallableStatement prepareCall(String sql, int nType, int nConcur, int nHo
      * @return true if statement pooling is disabled, false if it is enabled.
      */
     public boolean getDisableStatementPooling();
+
+    /**
+     * Returns the current flag value for useFmtOnly.
+     * 
+     * @return 'useFmtOnly' property value.
+     */
+    public boolean getUseFmtOnly();
+
+    /**
+     * Specifies the flag to use FMTONLY for parameter metadata queries.
+     * 
+     * @param useFmtOnly
+     *        boolean value for 'useFmtOnly'.
+     */
+    public void setUseFmtOnly(boolean useFmtOnly);

```

Comment:
can we be more consistent when describing this? getter says "useFmtOnly" and setter "FMTONLY"

Reply:
One returns a variable, and the variable in the driver is useFmtOnly. The other specifies it turns on a feature which uses the SQL Server "FMTONLY" feature to get metadata. Being consistent here loses information.